lib/internal/errors.js

```diff
@@ -1265,7 +1265,8 @@ E('ERR_INVALID_TUPLE', '%s must be an iterable %s tuple', TypeError);
 E('ERR_INVALID_URI', 'URI malformed', URIError);
 E('ERR_INVALID_URL', function(input) {
   this.input = input;
-  return `Invalid URL: ${input}`;
+  // Don't include URL in message (see #38614)
```

Comment:
```suggestion
  // Don't include URL in message.
  // (See https://github.com/nodejs/node/pull/38614)
```

nit: we usually include the full link in the comments